### ISSUE TITLE
Broken on HFW 4.86

### ISSUE THREAD

**Issue by RobertoD91** [DADi590/HAN-Toolbox#1]
Offline and online activation of HAN don't seem to work on HFW 4.86 through han toolbox.

Offline activation blocks the ps3 and requires a forced shutdown.

Online activation redirects to a page whose javascript refuses to proceed (I think it is not updated for HFW 4.86)

**Comment by Edw590** [comment]
Hey! Yep, the offline one I know it's broken (no idea if people go to Update Toolbox page or What's New on Browser, as that was the idea I had to warn people about that - no idea if you got to look there?). The online one is NOT supposed to be broken... It's from PS3Xploit site. Should work. I don't know why it doesn't. Hmm, is it possible for you to attach a picture of that here?

But yes, the offline one is broken and has been since I released that version. It seems it was a bug in PS3Xploit site and when I copied everything to update, I copied the bug too and let it on the toolbox since March... 2 days ago I started looking into the toolbox again. I've been terribly busy for these months and were unable to test the toolbox (it needs lots of testing to be stable and only now I'm on holidays). I think now it's all working except the Temperature Checker which seems to want more time to get ready (and I still need to check DEX mode of the toolbox, even though it's not much used, I think).

But thanks for the online part... I had no idea that wasn't working, and it should work. If you could have the picture it would be better for me to try to understand what's the problem. If it's really in the site, not much in my hands but I can tell them about it. If it's the redirection that it's not working, then I'll just have to change the link (should be working - it is on my 4.82 OFW). In any case, if none of them are working really, then it seems I gotta hurry haha. Thanks for the warning. Also, you're sure JavaScript is enabled on the browser, right?

**Comment by RobertoD91** [comment]
Sorry for the late reply.

Yes javascript is enabled, online button leads to http://ps3xploit.com/hfw/autoxml/han/han_enabler_auto.html then there is an alert "...you are on 4.86...not compatible..."

video:
https://youtu.be/eduQJz0yDiQ

**Comment by RobertoD91** [comment]
video will be removed in a while...there are some photo
![1](https://user-images.githubusercontent.com/298514/88739221-5ace4f80-d13a-11ea-9dce-e922a6137974.jpeg)
![2](https://user-images.githubusercontent.com/298514/88739218-57d35f00-d13a-11ea-9f42-81336fedba32.jpeg)
![3](https://user-images.githubusercontent.com/298514/88739153-29558400-d13a-11ea-9085-ec797c4843f8.jpeg)
![4](https://user-images.githubusercontent.com/298514/88739158-2c507480-d13a-11ea-8a8f-d31809920164.jpeg)
![5](https://user-images.githubusercontent.com/298514/88739163-2e1a3800-d13a-11ea-9b94-74e6e9877327.jpeg)

**Comment by Edw590** [comment]
No problem at all.

Right............. So I asked people to go to a broken option to fix the other broken option, amazing XD. Thank you. Well, then it means they didn't update the site - and they actually didn't, because the version of the tool is outdated. Actually the option on the toolbox is also broken because of 2 things: one, I'm on 4.82 and wasn't sure it was working or not but I trusted it was; and second, because the site had 2 wrong values on it and I copied them, so the option stopped working on 4.86 and I didn't realize that until people told me. But the link is correct. It's pointing to the right place. It's just not updated.

Anyways, it seems people haven't been able to use HAN then.... Cool... But the toolbox should be ready in a day or 2. I just added a new option to it (https://github.com/DADi590/HAN-Toolbox-Unofficial/releases, if you're interested in knowing) and I need to test it, and then test all the functions on DEX. On CEX they're all working already, supposedly, and that's good since most people don't even know what HAN DEX mode is and use mainly CEX.
